nlp/src/main/java/hivemall/nlp/tokenizer/KuromojiUDF.java

```diff
@@ -215,7 +215,7 @@ private static UserDictionary userDictionary(@Nullable final String[] userDictAr
         StringBuilder builder = new StringBuilder();
         for (String row : userDictArray) {
             builder.append(row);
-            builder.append("\r\n");
+            builder.append(System.getProperty("line.separator"));
```

Comment:
I guess `\n` is enough for IPA dict format. 

In some OS, it can be `\r` and it thus be malformed.